module/zcommon/zpool_prop.c

```diff
@@ -420,6 +420,9 @@ vdev_prop_init(void)
 	    boolean_na_table, sfeatures);
 
 	/* default index properties */
+	zprop_register_index(VDEV_PROP_FAILFAST, "failfast", B_TRUE,
+	    PROP_DEFAULT, ZFS_TYPE_VDEV, "on | off", "FAILFAST", boolean_table,
+	    sfeatures);
```

Comment:
Given that retries should be uncommon I think we should consider making the default `B_FALSE`.  This would be the most resilient behavior, of course the downside would be ZFS may not notice failing devices as promptly and proactively fault them.  Very slow device would still be easy to observe with `zpool status -s` or in the `zpool iostat` statistics.

Reply:
Thanks reasonable, fixed.